rust/operator-binary/src/spark_k8s_controller.rs

```diff
@@ -98,6 +101,13 @@ pub async fn reconcile(
         _ => None,
     };
 
+    if let Some(conn) = s3bucket.as_ref().and_then(|i| i.connection.as_ref()) {
+        if conn.tls.as_ref().is_some() {
+            tracing::warn!("The resource indicates S3-access should use TLS: TLS-verification has not yet been implemented \
```

Comment:
I personally would prefer erroring out if something other than WebPKI verification (which we currently always do) is specified for security reasons but thats personal preference ;)
Anyway i hope we support that soon ;)

Reply:
Something like 
```
         if let Some(tls) = &s3_conn.tls {
             match &tls.verification {
                 TlsVerification::None {} => return S3TlsNoVerificationNotSupportedSnafu.fail(),
                 TlsVerification::Server(server_verification) => {
                     match &server_verification.ca_cert {
                         CaCert::WebPki {} => {}
                         CaCert::SecretClass(secret_class) => return S3TlsCaVerificationNotSupportedSnafu.fail(),
                     }
                 }
             }
         }
```

Reply:
Implemented as suggested.